config/admins.txt

```diff
@@ -8,5 +8,5 @@
 # NOTE: syntax was changed to allow hyphenation of ranknames, since spaces are stripped.      #
 ###############################################################################################
 Purpose2 = Admin
-FlattestGuitar9 = Admin
+DaveTheHeadcrab = Admin
```

Comment:
🤔

Reply:
Oh, did not mean to commit this.
But I'll keep it.

Reply:
Protip: /tg/code already flags localhost connections as admin. Putting yourself in admins.txt is unnecessary